### PR TITLE
AT-SCE-UI 20: Clean button operation

### PR DESCRIPTION
**Pull request by Emiliano-Plascencia**
Added the "onclick" event in the clean button locaded in src/views/calculator.hbs 
Added the clean button function locaded in src/public/js/at-sce.util.js
Duplicate constant removed "sceServiceAPI"

### CODE REVIEW

**Review by moselGlez — APPROVED** [review]

File: src/public/js/at-sce.util.js [hunk 1]
```diff
@@ -212,3 +212,17 @@ function calculateProjectCost() {
...
+ 
+/*Reset Value Function*/
+function CleanOperation()
+{
+    document.getElementById("projectWillCostYou").innerHTML = resetvalue;
+    document.getElementById("costPerDay").innerHTML = resetvalue;
+    document.getElementById("costPerHour").innerHTML = resetvalue;
+    document.getElementById("taxesIVAandISR").innerHTML = resetvalue;
+    document.getElementById("charge").innerHTML = resetvalue;
+    document.getElementById("revenue").innerHTML = resetvalue;
```
Comment (at-internship) [inline src/public/js/at-sce.util.js]: we need to disable "Save" button after "clean" operation

File: src/views/calculator.hbs [hunk 2]
```diff
@@ -173,7 +173,8 @@
                     <div class="d-flex justify-content-around" style="margin: 0% 25% 5% 25%">
                          <button type="button" class="btn btn-primary btn-lg mx-2 px-4"
                               onclick="calculateProjectCost();">Calculate</button>
-                         <button type="reset" class="btn btn-outline-danger btn-lg mx-2 px-5">Clean</button>
+                         <button type="reset" class="btn btn-outline-danger btn-lg mx-2 px-5"
+                              onclick="CleanOperation();">Clean</button>
```
Comment (at-internship) [inline src/views/calculator.hbs]: you could use a function name that better describes the operation it performs

File: package.json [hunk 3]
```diff
@@ -21,7 +21,7 @@
   },
   "homepage": "https://github.com/at-internship/at-sce-ui#readme",
   "dependencies": {
-    "axios": "^0.20.0",
+    "axios": "^0.21.1",
```
Comment (at-internship) [inline package.json]: Why did you update the axios version?

**Review by at-internship — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (Emiliano-Plascencia) [reply package.json]: It's because I reinstalled the modules, I suppose it's the latest version of axios

Comment (at-internship) [reply package.json]: That is not necessary

**Comment by at-internship** [comment]
image

This branch is out of sync with **develop**. You need to integrate **develop**.

**Comment by Emiliano-Plascencia** [comment]
> image
> 
> This branch is out of sync with **develop**. You need to integrate **develop**.

**Comment by Emiliano-Plascencia** [comment]
Done, *CleanOperation* branch is already sync with *develop*

**Review by irvingesp2190 — APPROVED** [review]

File: package.json (hunk 3 above)
Comment (at-internship) [reply package.json]: should be axios": "^0.20.0",

File: src/public/js/at-sce.util.js [hunk 4]
```diff
@@ -191,3 +191,18 @@ function calculateProjectCost() {
 $(document).ready(function () {
   $("#historyTable").DataTable();
 });
+
+/*Constant reset value*/
+const resetvalue = "0.00";
+ 
+/*Reset Value Function*/
+function CleanFields()
```
Comment (at-internship) [inline src/public/js/at-sce.util.js]: `function cleanCalculatorForm()`

File: src/views/calculator.hbs [hunk 5]
```diff
@@ -174,7 +174,8 @@
                     <div class="d-flex justify-content-around" style="margin: 0% 25% 5% 25%">
                          <button type="button" class="btn btn-primary btn-lg mx-2 px-4"
                               onclick="calculateProjectCost();">Calculate</button>
-                         <button type="reset" class="btn btn-outline-danger btn-lg mx-2 px-5">Clean</button>
+                         <button type="reset" class="btn btn-outline-danger btn-lg mx-2 px-5"
+                              onclick="CleanFields();">Clean</button>
```
Comment (at-internship) [inline src/views/calculator.hbs]: `onclick="cleanCalculatorForm()"`

**Review by at-internship — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (Emiliano-Plascencia) [reply package.json]: Done

**Review by at-internship — APPROVED** [review]

**Pull request merged**